Protocol_zh.md

```diff
@@ -0,0 +1,182 @@
+# XDAG加密货币协议 
+#### Version 0.3.1 Apirl 6, 2018
+
+## 1.名词解释
+
+**区快地址** - 区块hash截取后的base64编码.
+
+**金额** - XDAG的数量，以Cheato为单位，类似比特币的Santoshi.  
+
+**区块** - Dagger的基本结构, 每个区块定长512字节, 结构定义在第二部分介绍.
+
+**链** - 前一个区块引用后一个区块区块，按一定的顺序组成链.  
+
+**Cheato** - 一个XDAG包括2^32个Cheato.  
+
+**Dagger创世块** - Unix格式：0x5A500000 ，也就是2018年1月6号 22:45:20 GMT.  
+
+**区块难度** - (2^128-1)/(hash_little / 2^160), hash_little = 小端字节序(hash(block)), hash_little大小为256 bits(32 bytes).
+
+**链难度** - 链上区块的难度之和.  
+
+**Hash** - 采用双sha256哈希算法, hash(block) = sha256(sha256(block)).
+
+**主区块** - 主区块链上的任何区块.  
+
+**主链** - 难度最大的链称为主链.  
+
+**链接** - 如果A具有类型2或3的字段，其中包含块B的截断散列，请参第二章. 
+
+**引用** - 如果从区块A到区块B是一段连续的区块组成，则称区块B被区块A引用.
+
+**i-引用** - 如果区块A的第i个域连接到区块C，并且区块C引用区块B，则称区块B是区块A的i-引用.  
+
+**分离链** - 链上的区块属于不同的时间片.  
+
+**最短链** - 一个从区块A到区块B的链，每个区块都是i-引用B。i越小，链越短.  
+
+**时间片** - 每64秒一个时间片，时间为Unix格式，低6bits等于0.  
+
+**区块时间** - 区块时间记录在区块头信息里，从1970年1月1号开始，单位是1/1024秒.  
+
+**交易** - 交易即区块.  
+
+**hash截断** - hash小端字节序表示，取最低24个字节.
+
+
+## 2.区块
+
+一个区块有16个字段，每个字段32个字节，每个字段type可能为0到15，一般区块，第0个字段type必须是1;伪块，第0个字段必须是类型0。有关传输用的伪块，详看第5章
+
+字段的类型0-15：
+
+	0		随机数, 32字节.  
+
+	1		区块头. 结构如下:
+			- 64-bit 传输层头, 在计算区块散列的时候，这个头信息要全部设置为0;
+			- 64-bit 小端字节序, 包括字段类型0..15. 每个类型4 bits. 字段0的类型编码为低4位.
+			- 64-bit 小端字节序, 区块的时间.
+			- 64-bit 小端字节序, 交易的手续费.
+
+	2		交易的输入. 连接到另外一个区块B.  
+			结构如下:
+			- 192-bit 区块B的hash截断.
+			- 64-bit 小端字节序, 区块被应用后的余额.
+
+
+	3		交易的输出. 连接到其他区块. 
+	        结构如下:
+			- 192-bit 区块B的hash截断.
+			- 64-bit 小端字节序, 区块被应用后的余额.
```

Comment:
- 64-bit little-endian number, **the amount that is taken from the balance of block B** then the transaction is applied.

It should be amount, not balance in translation.